website/docs/r/nginx_deployment.html.markdown

```diff
@@ -154,6 +158,16 @@ A `network_interface` block supports the following:
 
 * `subnet_id` - (Required) Specify The SubNet Resource ID to this Nginx Deployment.
 
+---
+
+An `auto_scale_profile` block exports the following:
+
+* `name` - (Required) Specify the name of the autoscaling profile.
+
+* `min` - (Required) Specify the minimum number of NGINX capacity units for this NGINX Deployment.
+
+* `max` - (Required) Specify the maximum number of NGINX capacity units for this NGINX Deployment.
```

Comment:
In the resource documentation for blocks with arguments we use the phrase `supports the following` instead of `exports the following`
```suggestion
An `auto_scale_profile` block supports the following:

* `name` - (Required) Specify the name of the autoscaling profile.

* `min` - (Required) Specify the minimum number of NGINX capacity units for this NGINX Deployment.

* `max` - (Required) Specify the maximum number of NGINX capacity units for this NGINX Deployment.
```

Reply:
done 👍

Reply:
Going to resolve this as the suggestion was addressed.